scopes/ui-foundation/use-box/tab/tab.spec.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { TabWithText, TabWithChildElement, ActiveTab } from './tab.compositions';
+
+describe('basic tab', () => {
+  it('should render with the correct text', () => {
+    const { getByText } = render(<TabWithText />);
+    const rendered = getByText('bit');
+    expect(rendered).toBeTruthy();
+  });
+
+  it('should render without active class', () => {
+    const { getByText } = render(<TabWithText />);
+    const rendered = getByText('bit');
+    expect(rendered.classList.contains('active')).not.toBeTruthy();
```

Comment:
might be able to do `expect(rendered.classList).not.toHave('active')`, or something similar (`not.toInclude('active')`)